docs/farming-&-staking/farming/intro.mdx

```diff
@@ -41,8 +42,8 @@ For those who prefer a visual guide, we have prepared installation videos to ass
 
 | Hardware | CPU | RAM | Node Storage | Farm Storage |
 | --- | --- | :-: | :-: | :-: |
-| Minimum [¹](#min-hardware) | 4 Cores+ | 8GB | 100GB [²](#min-node-storage) | 100GB |
-| Recommended | Intel Core i7-6700 or equivalent | 16GB+ | 256GB | ♾️ [³](#farm-storage) |
+| Minimum [¹](https://docs.autonomys.xyz/farming/intro#min-hardware) | 4 Cores+ | 8GB | 100GB [²](https://docs.autonomys.xyz/farming/intro#min-node-storage) | 100GB |
```

Comment:
These links should lead the absolute path to the internal file, not its deployed version

Reply:
This is a workaround because it appears that Docusaurus has an issue with hidden anchors, resulting in build errors related to broken links, even though they are valid and functional.

We can either keep it as is, accept the build warnings, or we can refactor the page to avoid using footnotes like this. If we use traditional footnotes, they are automatically generated at the footer of the page, not in the hardware section as they currently are.

Reply:
I see, okay, let's keep it as is then, thanks for clarifying!